app/src/pages/instance/allUsersPage/allUsersHeader/createUserModal/createUserModal.scss

```
@@ -16,6 +16,10 @@
 
 .modal {
   width: 720px;
+  :global(.rc-scrollbars-view) {
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)